### PR TITLE
Inwebo Authenticator change to IS 5.1.0-Beta

### PR DESCRIPTION
**Pull request by ashalya**

### CODE REVIEW

File: inwebo/inwebo-authenticator/org.wso2.carbon.identity.authenticator/src/main/java/org/wso2/carbon/identity/authenticator/InweboAuthenticator.java [hunk 1]
```diff
@@ -0,0 +1,225 @@
...
+
+    /**
+     * Check whether the authentication or logout request can be handled by the authenticator
+     */
+    @Override
+    public boolean canHandle(HttpServletRequest request) {
+        if (log.isDebugEnabled()) {
+            log.debug("Inside InweboAuthenticator.canHandle()");
+        }
+        return (!StringUtils.isEmpty(request.getParameter("inwebo")));
```
Comment (keerthu) [inline inwebo/inwebo-authenticator/org.wso2.carbon.identity.authenticator/src/main/java/org/wso2/carbon/identity/authenticator/InweboAuthenticator.java]: Use StringUtils.isNotEmpty() rather using !StringUtils.isEmpty().

File: inwebo/inwebo-authenticator/org.wso2.carbon.identity.authenticator/src/main/java/org/wso2/carbon/identity/authenticator/InweboAuthenticator.java [hunk 2]
```diff
@@ -0,0 +1,225 @@
...
+        if (authenticatorProperties != null) {
+            String serviceId = authenticatorProperties.get(InweboConstants.SERVICE_ID);
+            String p12file = authenticatorProperties.get(InweboConstants.INWEBO_P12FILE);
+            String p12password = authenticatorProperties.get(InweboConstants.INWEBO_P12PASSWORD);
+            if (!StringUtils.isEmpty(authenticatorProperties.get(InweboConstants.RETRY_COUNT))) {
+                waitTime = Integer.parseInt(authenticatorProperties.get(InweboConstants.RETRY_COUNT));
+            } else {
+                waitTime = Integer.parseInt(InweboConstants.WAITTIME_DEFAULT);
+            }
+            if (!StringUtils.isEmpty(authenticatorProperties.get(InweboConstants.RETRY_INTERVAL))) {
```
Comment (keerthu) [inline inwebo/inwebo-authenticator/org.wso2.carbon.identity.authenticator/src/main/java/org/wso2/carbon/identity/authenticator/InweboAuthenticator.java]: Same as above comment.

File: inwebo/inwebo-authenticator/org.wso2.carbon.identity.authenticator/src/main/java/org/wso2/carbon/identity/authenticator/InweboAuthenticator.java (hunk 1 above)
Comment (ashalya) [inline inwebo/inwebo-authenticator/org.wso2.carbon.identity.authenticator/src/main/java/org/wso2/carbon/identity/authenticator/InweboAuthenticator.java]: fixed in https://github.com/wso2/is-connectors/pull/52

File: inwebo/inwebo-authenticator/org.wso2.carbon.identity.authenticator/src/main/java/org/wso2/carbon/identity/authenticator/InweboAuthenticator.java (hunk 2 above)
Comment (ashalya) [inline inwebo/inwebo-authenticator/org.wso2.carbon.identity.authenticator/src/main/java/org/wso2/carbon/identity/authenticator/InweboAuthenticator.java]: fixed in https://github.com/wso2/is-connectors/pull/52

**Pull request merged**